web/src/components/EventListItem.tsx

```diff
@@ -18,10 +24,38 @@ import {
 } from '../assets'
 import { EXCERPT_MAX_CHARS } from '../constants'
 import useWindowDimensions from '../hooks/useWindowDimensions'
-import ListItem from './ListItem'
+import Link from './base/Link'
 
-const Content = styled('div')`
-  overflow-wrap: anywhere;
+const StyledListItem = styled(ListItem)`
+  [class*='MuiListItemSecondaryAction-root'] {
```

Comment:
```suggestion
  .MuiListItemSecondaryAction-root {
```
Or simply like that 🙃